src/handler.ts

```diff
@@ -338,6 +339,7 @@ export namespace Scheduler {
     file_extensions: string[];
     output_formats: IOutputFormat[];
     metadata: { [key: string]: any };
```

Comment:
```suggestion
    metadata: { [key: string]: string };
```

Reply:
Could any metadata item be, for example, an array?

Reply:
No, the intent of metadata was to provide simple values for display.